packages/core/src/components/sections/Incentives/Incentives.tsx

```diff
@@ -1,17 +1,18 @@
-import UIIncentives from 'src/components/ui/Incentives/Incentives'
 import type { Incentive } from 'src/components/ui/Incentives'
+import UIIncentives from 'src/components/ui/Incentives/Incentives'
 
 import Section from '../Section'
 import styles from './section.module.scss'
 
 interface Props {
   incentives: Incentive[]
+  label?: string
 }
 
-function Incentives({ incentives }: Props) {
+function Incentives({ incentives, label }: Props) {
   return (
     <Section className={`${styles.section} section-incentives layout__section`}>
-      <UIIncentives incentives={incentives} colored />
+      <UIIncentives incentives={incentives} colored label={label ?? ''} />
```

Comment:
Leaving it as an empty string isn’t ideal, but it works for now. Ideally, we should receive a label from the CMS (if applicable) to identify which Incentive section we’re referring to. 😞 

The issue occurs when we have more than one Incentive section on the same page, as we currently lack a label reference to differentiate between them (this is the case on the homepage). 



<img width="406" alt="Screenshot 2024-09-16 at 11 55 49" src="https://github.com/user-attachments/assets/5fd83d9b-f94f-4be6-97d2-d1be33ed9a6f">